### PR TITLE
Add word translation & hint

### PR DESCRIPTION
**Pull request by olyakolesnichenko**

### CODE REVIEW

File: client/src/locales/en/translation.json [hunk 1]
```diff
@@ -2,5 +2,8 @@
   "Hello": "Hello",
   "Do you know this word?": "Do you know this word?",
   "YES": "YES",
-  "NO": "NO"
+  "NO": "NO",
+  "Show next": "Show next word >",
+  "Give the cue": "Give the cue"
+
```
Comment (o-vitaliy) [inline client/src/locales/en/translation.json]: Could you double-check if this file formatted?

File: server/server.js [hunk 2]
```diff
@@ -1,11 +1,31 @@
...
+    {"word":'liberty', "translation": 'свобода'},
+    {"word":'tranquility', "translation": 'спокойствие'}];
+  res.status(200).json({word: words[Math.floor(Math.random()*words.length)]});
+});
+app.post('/answer', function (req, res) {
+  const answerData = req.body;
+console.log(req.body);
+  // if (answerData.word) {
+    res.status(200).json({result: true});
+  // } else {
```
Comment (o-vitaliy) [inline server/server.js]: Could you remove the commented code?

File: client/src/locales/en/translation.json (hunk 1 above)
Comment (olyakolesnichenko) [reply client/src/locales/en/translation.json]: done

File: server/server.js (hunk 2 above)
Comment (olyakolesnichenko) [reply server/server.js]: done

**Review by o-vitaliy — APPROVED** [review]

**Pull request merged**